### ISSUE TITLE
Orbot Android Device Rotation Bug

### ISSUE THREAD

**Issue by ivanjx** [guardianproject/orbot#376]
Hi,

I have been using orbot for a few days now. My device is not rooted. I set the orbot to use vpn mode full device. All works fine i can connect to my server with ssh via tor network, browse with chrome within tor network etc. But as soon as i rotate my device (like watching full screen video in landscape mode, turning off device rotation lock) the tor connection seems to be suddenly  disconnected. However the app itself says that it is still connected. To try on your android just connect in orbot then after a few moment just rotate the device. My device is samsung galaxy s10 with android 10 running.

Thanks.

**Comment by realivanjx** [comment]
Here is the log from my device:

```
INFO: entry_guard_inc_circ_attempt_count: Got success count 65.000000/68.000000 for guard dragonhoard ($24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5) 
INFO: circuit_finish_handshake: Finished building circuit hop: 
INFO: exit (high-uptime) circ (length 3, last hop PacketPusher): $24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5(open) $E5F1B4995E939C13D4DEE7B13112A9B10FDB61A0(open) $9ECEC047BA0596C51EDD098616C8A86697A12824(closed) 
INFO: circuit_send_intermediate_onion_skin: Sending extend relay cell. 
INFO: handle_relay_cell_command: 23: end cell (closed normally) for stream 20598. Removing stream. 
StreamStatus (45): CLOSED
INFO: circuit_predict_and_launch_new: Have 9 clean circs (3 internal), need another exit circ. 
INFO: choose_good_exit_server_general: Found 1134 servers that might support 0/0 pending connections. 
INFO: choose_good_exit_server_general: Chose exit server '$B1045E12FA4EA0D457A74013866CB41DC0D290BF~broccoli at 109.70.100.16 and [2a03:e600:100::10]' 
INFO: extend_info_from_node: Including Ed25519 ID for $B1045E12FA4EA0D457A74013866CB41DC0D290BF~broccoli at 109.70.100.16 and [2a03:e600:100::10] 
INFO: select_primary_guard_for_circuit: Selected primary guard dragonhoard ($24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5) for circuit. 
INFO: extend_info_from_node: Including Ed25519 ID for $24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5~dragonhoard at 159.69.21.196 and [2a01:4f8:1c1c:2e49::1] 
INFO: extend_info_from_node: Including Ed25519 ID for $4452EEE635A29539EE0C120B1A417D1E2C812F12~Unnamed at 51.107.68.158 
INFO: circuit_send_first_onion_skin: First hop: finished sending CREATE cell to '$24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5~dragonhoard at 159.69.21.196 and [2a01:4f8:1c1c:2e49::1]' 
INFO: circuit_finish_handshake: Finished building circuit hop: 
INFO: exit (high-uptime) circ (length 3, last hop PacketPusher): $24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5(open) $E5F1B4995E939C13D4DEE7B13112A9B10FDB61A0(open) $9ECEC047BA0596C51EDD098616C8A86697A12824(open) 
INFO: entry_guards_note_guard_success: Recorded success for primary confirmed guard dragonhoard ($24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5) 
INFO: circuit_build_no_more_hops: circuit built! 
INFO: pathbias_count_build_success: Got success count 66.000000/68.000000 for guard dragonhoard ($24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5) 
INFO: circuit_finish_handshake: Finished building circuit hop: 
INFO: exit (high-uptime) circ (length 3, last hop broccoli): $24A17A4C1FA8C2108ACDE3BE681B0731384BF3A5(open) $4452EEE635A29539EE0C120B1A417D1E2C812F12(closed) $B1045E12FA4EA0D457A74013866CB41DC0D290BF(closed) 
INFO: circuit_send_intermediate_onion_skin: Sending extend relay cell. 
INFO: handle_relay_cell_command: 27: end cell (closed normally) for stream 25721. Removing stream. 
StreamStatus (54): CLOSED
INFO: handle_relay_cell_command: 17: end cell (closed normally) for stream 61509. Removing stream. 
StreamStatus (44): CLOSED
INFO: or_state_save: Saved state to "/data/data/org.torproject.android/tordata/state" 
INFO: channelpadding_send_padding_cell_for_callback: Sending netflow keepalive on 1 to [scrubbed] ([scrubbed]) after 13440 ms. Delta 0ms 
INFO: handle_relay_cell_command: 26: end cell (closed normally) for stream 20599. Removing stream. 
StreamStatus (46): CLOSED
```

**Comment by bitmold** [comment]
Closing since this is so old and there are other rotation bugs...
